src/javascript/app_2/pages/trading/components/purchase.jsx

```diff
@@ -32,36 +51,59 @@ const Purchase = ({
     trade_types,
 }) => (
     Object.keys(trade_types).map((type, idx) => {
-        const info = proposal_info[type] || {};
+        const info         = proposal_info[type] || {};
         const is_logged_in = isLoggedIn();
+        const el_currency  = <span className={`symbols ${(Client.get('currency') || 'usd').toLowerCase()}`}/>;
```

Comment:
can we use `USD` in all caps, so making it lowercase makes sense? :P

Reply:
Sure :)